ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-commands.sh

```diff
@@ -0,0 +1,565 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function warn_0_case_executed {
+    local count
+    count="$(ls ${ARTIFACT_DIR} | wc -l)"
+    if [ $((count)) == 0 ] ; then
+        mkdir --parents "${ARTIFACT_DIR}"
+        cat >"${ARTIFACT_DIR}/junit-ginkgo-result.xml" <<- EOF
+<testsuite name="openshift-extended-test" tests="1" errors="1">
+  <testcase name="Overall status of openshift-extended test">
+    <failure message="">Caution: NO test cases executed.</failure>
+  </testcase>
+</testsuite>
+EOF
+
+    fi
+}
+
+function save_oidc_tokens {
+    echo "Saving oidc tokens back to SHARED_DIR"
+    cp "$token_cache_dir"/* "$SHARED_DIR"/oc-oidc-token
+    ls "$token_cache_dir" > "$SHARED_DIR"/oc-oidc-token-filename
+}
+
+function exit_trap {
+    echo "Exit trap triggered"
+    date '+%s' > "${SHARED_DIR}/TEST_TIME_TEST_END" || :
+    warn_0_case_executed
+    if [[ -r "$SHARED_DIR/oc-oidc-token" ]] && [[ -r "$SHARED_DIR/oc-oidc-token-filename" ]]; then
+        save_oidc_tokens
+    fi
+}
+
+trap 'exit_trap' EXIT
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json
+export GCP_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/gce.json
+export HOME=/tmp/home
+export PATH=/usr/local/go/bin:/usr/libexec/origin:/opt/OpenShift4-tools:$PATH
+export REPORT_HANDLE_PATH="/usr/bin"
+export ENABLE_PRINT_EVENT_STDOUT=true
+
+# add for hosted kubeconfig in the hosted cluster env
+if test -f "${SHARED_DIR}/nested_kubeconfig"
+then
+    export GUEST_KUBECONFIG=${SHARED_DIR}/nested_kubeconfig
+fi
+
+# restore external oidc cache dir for oc
```

Comment:
@geliu2016 
please move 
```shell
# restore external oidc cache dir for oc
if [[ -r "$SHARED_DIR/oc-oidc-token" ]] && [[ -r "$SHARED_DIR/oc-oidc-token-filename" ]]; then
    echo "Restoring external OIDC cache dir for oc"
    export KUBECACHEDIR
    KUBECACHEDIR="/tmp/output/oc-oidc"
    token_cache_dir="$KUBECACHEDIR/oc"
    mkdir -p "$token_cache_dir"
    cat "$SHARED_DIR/oc-oidc-token" > "$token_cache_dir/$(cat "$SHARED_DIR/oc-oidc-token-filename")"
    oc whoami
fi
```
after
```shell
# setup proxy
if test -f "${SHARED_DIR}/proxy-conf.sh"
then
    source "${SHARED_DIR}/proxy-conf.sh"
fi
```

refer to https://github.com/openshift/release/pull/52452

Reply:
Updated.